functions/ingest/format/defaulttsdb.go

```diff
@@ -57,8 +58,18 @@ type defaultTsdb struct{}
 func (Ingester defaultTsdb) Ingest(tsdbAppender tsdb.Appender, event nuclio.Event) interface{} {
 	var requests []request
 
-	if err := json.Unmarshal(event.GetBody(), &requests); err != nil {
-		InternalError(errors.Wrap(err, "Failed to deserialize JSON").Error())
+	body := event.GetBody()
+	if strings.HasPrefix(strings.TrimSpace(string(body)), "[") {
```

Comment:
i guess the body might be big if there are a lot of different labelsets in a request, why not do something like:
`string(body[0]) == "["`